integration-tests/manifests/sample-daemonset.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: fluent-bit
+  labels:
+    app: fluent-bit
+spec:
+  selector:
+    matchLabels:
+      name: fluent-bit
+  template:
+    metadata:
+      labels:
+        name: fluent-bit
+    spec:
+      containers:
+        - name: fluent-bit
+          image: nginx:latest
```

Comment:
Why does a fluent-bit container run an nginx image?

Reply:
I thought we needed the same image for all the sample-apps which is why I did this.